browser/src/ClinvarVariantsTrack/ClinvarVariantTrack.tsx

```diff
@@ -119,14 +135,16 @@ const ClinvarVariantTrack = ({ referenceGenome, transcripts, variants }: Props)
 
   const [showOnlyGnomad, setShowOnlyGnomad] = useState(false)
   const [isExpanded, setIsExpanded] = useState(false)
+  const [starFilter, setStarFilter] = useState(0)
 
   const filteredVariants = variants.filter(
     (v) =>
       // @ts-expect-error TS(7053) FIXME: Element implicitly has an 'any' type because expre... Remove this comment to see the full error message
       includedClinicalSignificanceCategories[clinvarVariantClinicalSignificanceCategory(v)] &&
       // @ts-expect-error TS(7053) FIXME: Element implicitly has an 'any' type because expre... Remove this comment to see the full error message
       includedConsequenceCategories[getCategoryFromConsequence(v.major_consequence)] &&
-      (!showOnlyGnomad || v.in_gnomad)
+      (!showOnlyGnomad || v.in_gnomad) &&
+      (starFilter === 0 || v.gold_stars >= starFilter)
```

Comment:
The value of `v.gold_stars` should always be non-negative, and hence if `starFilter` is 0, `v.gold_stars >= starFilter` will always be true. Hence you can get rid of the explicit `starFilter === 0` check here.